product/views.py

```diff
@@ -1,3 +1,50 @@
-from django.shortcuts import render
+import json
+from django.http        import JsonResponse
+from django.views       import View
+from .models            import FilteringColor, Product, SubCategory, Series, Sex, Color, ProductSize, Size, Medium, ProductInformation 
 
+class DetailView(View):
```

Comment:
상세페이지 View를 잘 작성해 주셨습니다~
다만, 앤드포인트는 페이지를 기준으로 만드는 것이 아니라, 기능 기준으로 만들어 주셔야 합니다.

상세페이지를 기준으로 view를 작성할 경우에, 해당 앤드포인트는 상세페이지에서 밖에 사용할 수 없습니다!
그렇기 때문에 기능 기준으로 앤드포인트를 작성해 주셔야 합니다.

```suggestion
class ProductView(View):
```

이렇게 작성하게되면, 상품 상세페이지를 요청하는 것이 아니라 상품 데이터를 요청하는 의미를 갖게 됩니다.
그렇게 때문에 상품 정보가 필요한 여러 곳에서 해당 앤드포인트를 사용할 수 있습니다!

Reply:
감사합니다!!